dynamic_stack_decider/tests/test_parser.py

```diff
@@ -98,7 +97,11 @@ def test_multiple_subbehavior_references(self):
 
 
 if __name__ == '__main__':
-    with open('report.xml', 'wb') as output:
-        unittest.main(
-            testRunner=xmlrunner.XMLTestRunner(output=output)
-        )
+    try:
+        import xmlrunner
+        with open('report.xml', 'wb') as output:
+            unittest.main(
+                testRunner=xmlrunner.XMLTestRunner(output=output)
+            )
+    except ImportError:
+        unittest.main()
```

Comment:
This is not strictly speaking part of the stack element reimplementation but I think we can let it pass ;)

Reply:
Thank you :)